advocacy_docs/partner_docs/ToadEdgeGuide/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Partner Information'
+description: 'Details for Quest and Toad Edge'
+
+---
+| &nbsp;     | &nbsp; |
+| ----------- | ----------- |
+| **Partner Name**      | Quest       |
+| **Partner Product**   | Toad Edge      |
+| **Web Site**      | https://www.quest.com/products/toad-edge/     |
+| **Version**   | Toad Edge 2.4.1   |
+| **Product Description**      |   Flexible toolset for database development and management tasks    |
```

Comment:
Description.
Toad Edge is a lightweight, desktop toolset that simplifies the development and management of open source relational databases. Toad Edge makes it easy for you to ramp-up quickly, with support for coding, editing, schema compare and sync, and DevOps CI processes.